CONTRIBUTING.md

```diff
@@ -2,117 +2,63 @@
 
 - [Contributing](#contributing)
   - [Architecture Decision Records (ADR)](#architecture-decision-records-adr)
+  - [Dependencies](#dependencies)
+  - [Protobuf](#protobuf)
+  - [Forking](#forking)
+  - [Developing and testing](#developing-and-testing)
   - [Pull Requests](#pull-requests)
+    - [PR Targeting](#pr-targeting)
     - [Process for reviewing PRs](#process-for-reviewing-prs)
+    - [PR merge procedure](#pr-merge-procedure)
     - [Updating Documentation](#updating-documentation)
-  - [Forking](#forking)
-  - [Dependencies](#dependencies)
-  - [Protobuf](#protobuf)
-  - [Testing](#testing)
   - [Branching Model and Release](#branching-model-and-release)
-    - [PR Targeting](#pr-targeting)
-    - [Development Procedure](#development-procedure)
-    - [Pull Merge Procedure](#pull-merge-procedure)
-    - [Release Procedure](#release-procedure)
-    - [Point Release Procedure](#point-release-procedure)
+    - [New major release branch](#new-major-release-branch)
+    - [New minor release branch](#new-minor-release-branch)
+    - [Point release procedure](#point-release-procedure)
+    - [Post-release procedure](#post-release-procedure)
```

Comment:
I think needing to have a table of contents means we have too much information in this document, in my opinion. I think the shorter the document, the more likely folks will be to read all of it